app/main.py

```diff
@@ -1,3 +1,2 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    # write your code here
-    pass
+    return sum([1 for i in phrase if letter.lower() == i.lower()])
```

Comment:
Consider using a generator expression instead of a list comprehension for counting occurrences. This can be done by removing the square brackets: `sum(1 for i in phrase if letter.lower() == i.lower())`. This change can improve performance slightly by avoiding the creation of an intermediate list.